Add upgrade notes for the second parameter in getByPath() method (#13046)
pimcore/pimcore: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

    security:
        enable_authenticator_manager: true
    ```

> - [Elements] Calling the methods `Asset::getById()`, `Document::getById()`, `DataObject::getById()`, `Asset::getByPath()`, `Document::getByPath()` and `DataObject::getByPath()` with second boolean parameter `$force` is deprecated and will throw exception in Pimcore 11. Calling the method `Element\Service::getElementById::getElementById()` with third boolean parameter `$force` is also deprecated and will throw exception in Pimcore 11. Instead pass this parameter as associative array with `$force` value.

Reviewer: ```suggestion
- [Elements] Calling the methods `getById` and `getByPath` on `Asset`,`Document`,`DataObject` with second boolean parameter `$force` and `Element\Service::getElementById` with third boolean  parameter `$force` is deprecated and will throw exception in Pimcore 11. Instead pass the parameter as associative array with `$force` value.
```
